rocketpy/Flight.py

```diff
@@ -512,6 +514,7 @@ def __init__(
         self,
         rocket,
         environment,
+        railLength,
         inclination=80,
         heading=90,
```

Comment:
I think it is totally valid to change rail_length to be optional and to its default value be `None`. Is the Flight class already structured for this change? Or would not having `railLength` break it?

Even better question, what happens when `railLength = 0`?

Also should inclination and heading have `None` as default too?

Reply:
> I think it is totally valid to change rail_length to be optional and to its default value be `None`. Is the Flight class already structured for this change? Or would not having `railLength` break it?

I tried but the flight class was raising problem with some weird ZeroDivisionError.
I think it is just easier if we move with this PR without having a standard value for the railLength, and maybe in the future we change it in another PR, this shouldn't create any break changes.

> Even better question, what happens when `railLength = 0`?

A ValueError will be raised if a rail length <= 0 is passed, just see the last commits.

> Also should inclination and heading have `None` as default too?
Not in the scope of this PR. Great suggestion though, let's address this in another PR in the future, for sure.